src/libs/actions/Policy.js

```diff
@@ -741,17 +746,14 @@ function generateDefaultWorkspaceName(email = '') {
         return defaultWorkspaceName;
     }
 
-    // Check if this name already exists in the policies
-    let suffix = 0;
-    _.forEach(allPolicies, (policy) => {
-        const name = lodashGet(policy, 'name', '');
-
-        if (name.toLowerCase().includes(defaultWorkspaceName.toLowerCase())) {
-            suffix += 1;
-        }
-    });
-
-    return suffix > 0 ? `${defaultWorkspaceName} ${suffix}` : defaultWorkspaceName;
+    // find default named workspaces and increment the last number
+    const numberRegEx = new RegExp(`${defaultWorkspaceName} ?(\\d*)`, 'i');
+    const defaultWorkspaceNumbers = _.chain(allPolicies)
+        .filter(policy => policy.name && numberRegEx.test(policy.name))
+        .map(policy => parseInt(numberRegEx.exec(policy.name)[1] || 1, 10)) // parse the number at the end
+        .value();
+    const lastNumber = defaultWorkspaceNumbers.sort().pop();
+    return lastNumber ? `${defaultWorkspaceName} ${lastNumber + 1}` : defaultWorkspaceName;
```

Comment:
Does not increment its always the same, it's broken now.

Delete all workspace create multiple workspaces all are same

Reply:
I'm not seeing this in my test. Did you pull the latest changes on the branch?

https://user-images.githubusercontent.com/6833644/197302648-0ba8f7ee-d96b-49b2-89a5-556b9e261802.mov

Reply:
Yes did,

https://user-images.githubusercontent.com/85645967/197349643-7054e686-08b2-42b3-8fe5-25e9450049e2.mov

Reply:
Okay the public gmail domain and the `+` in your user email is what caused the conflict. I was able to reproduce with `test+123@gmail.com` and fix it